app/repos/mappers/cdo.js

```diff
@@ -55,6 +57,75 @@ const mapSummaryCdoDaoToDto = (summaryCdo) => {
   }
 }
 
+const mapCdoPersonToPerson = (person) => {
```

Comment:
There is probably not an easy way round this but your are essentially duplicating the DTO code for these objects

Reply:
Yes, this was by choice.  By having it as a separate function, means they can divurge from each other if we do more changes.